Makefile

```diff
@@ -414,7 +414,7 @@ endif
 setup-go-work: export BUILD_ENTERPRISE_READY := $(BUILD_ENTERPRISE_READY)
 setup-go-work: export BUILD_BOARDS := $(BUILD_BOARDS)
 setup-go-work: ## Sets up your go.work file
-	./scripts/setup_go_work.sh
+	./scripts/setup_go_work.sh $(IGNORE_IF_EXISTS)
```

Comment:
Is `IGNORE_IF_EXISTS` unique to this target, or would it have application to other makefile targets? Wondering if the name should be more explicit, ala `IGNORE_GO_WORK_IF_EXISTS` or something.

Reply:
It's global, but I didn't want to make it longer than it already is. Although considering that it won't be used normally except a select few, I can change it.